packages/overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -149,9 +149,7 @@ export const PositionMixin = (superClass) =>
       window.visualViewport.addEventListener('resize', this._updatePosition);
       window.visualViewport.addEventListener('scroll', this.__onScroll, true);
 
-      this.__positionTargetAncestorRootNodes = getAncestorRootNodes(this.positionTarget).filter(
-        (node) => node !== document,
-      );
+      this.__positionTargetAncestorRootNodes = getAncestorRootNodes(this.positionTarget);
```

Comment:
This restores listening to `scroll` events on the `document` for desktop browsers.